mango_time_series/mango_time_series/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+Main file
+"""
+
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)
+__version__ = "0.2.3a1"
```

Comment:
version = "0.0.1a2"?

Reply:
Should be 0.2.0a1 at least? So we can setup dependecies in the monorepo as discussed